moveit_planners/stomp/stomp_moveit/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package>
+  <name>stomp_moveit</name>
+  <version>0.0.0</version>
+  <description>The stomp_moveit package</description>
+
+  <maintainer email="jrgnichodevel@gmail.com">Jorge Nicho</maintainer>
+  <license>Apache 2.0</license>
+  <author email="jrgnichodevel@gmail.com">Jorge Nicho</author>
```

Comment:
This entire package and all the code in it was created from scratch although it was inspired by the original work done by the willow staff.